imports/plugins/core/orders/client/templates/orders.js

```diff
@@ -89,22 +79,29 @@ const OrderHelper =  {
 
 Template.orders.onCreated(function () {
   this.state = new ReactiveDict();
+  this.orderLimits = new ReactiveDict();
   this.state.setDefault({
     orders: []
   });
+  this.orderLimits.setDefault({
+    new: 2,
```

Comment:
Yeah, I'd set it to 2 for my own testing purposes, but in production people would have a lot more orders. What would you suggest the number be?